### PR TITLE
Fixes annoying qr popup on parsing LNURL withdraw receive in wallet

### PR DESCRIPTION
**Pull request by arcbtc**

### CODE REVIEW

**Comment by dni** [comment]
i cannot really tell what the issue is, can you provide a screenshot or something what the problem is?

**Review by dni — APPROVED** [review]
still going to approve it, despite missing some context, seems simple enough

File: lnbits/core/templates/core/wallet.html [hunk 1]
```diff
@@ -569,7 +569,10 @@ <h6 class="text-subtitle1 q-mt-none q-mb-sm">
             ></q-spinner>
           </q-form>
         </q-card>
-        <q-card v-else class="q-pa-lg q-pt-xl lnbits__dialog-card">
+        <q-card
+          v-else-if="receive.paymentReq && receive.lnurl == null"
```
Comment (motorina0) [inline lnbits/core/templates/core/wallet.html:573]: the corresponding `v-if` is:
```python
        <q-card
          v-if="!receive.paymentReq"
```
so `receive.paymentReq` is not required in this condition

Comment (motorina0) [inline lnbits/core/templates/core/wallet.html:573]:
```suggestion
          v-else-if="receive.lnurl == null"
```

Comment (motorina0) [reply lnbits/core/templates/core/wallet.html:573]: what if this condition is false: `receive.lnurl == null` (`receive.lnurl != null`)?

Comment (arcbtc) [reply lnbits/core/templates/core/wallet.html:573]: The invoice QR code should only fire creating an invoice, not when an invoice is parsed out of an lnurl-withdraw

**Review by motorina0 — APPROVED** [review]

**Review by talvasconcelos — APPROVED** [review]

**Pull request merged**